### ISSUE TITLE
2TB WD hard drive, front panel fried and disposed off, 

### ISSUE THREAD

**Issue by ahmedkpasha** [themaddoctor/linux-mybook-tools#15]
Dear Thomas, i have a 2TB WD hard drive, the front panel was fried, I didn't know that it contained software to access the drive, I threw it away, now I have the drive, but when I attached it to my PC, it shows 1.9TB of un-allocated space, so i partitioned & formatted it into 4 further drives, now I tried to run a Data recovery software, but no file was found. I am guessing its because of the password that I put on it.

The proof that it is mine can be given. I even know the password, just that I don't know where to put it and how to access my files, these are files from when I first owned a Laptop. The data there i not just mine, but My brothers and sisters asked me to back-up their PCs, It means the world to all of us.

Could you please take some time out and help me in this regard?

**Comment by themaddoctor** [comment]
Partitioning and formatting it was a big mistake. Even if you can recover the encryption key, you will have to do data recovery again on the decrypted drive.

Start by downloading the "findkeyblock.sh" script from this project and look in the PDF file for the command to run it. It's something like

``` sudo bash findkeyblock.sh /dev/sdc ```

You have to be running linux, of course.

**Comment by ahmedkpasha** [comment]
God help me, Will try to do that thomas, will let you know how it goes....

**Comment by ahmedkpasha** [comment]
where exactly do I run this code, I just booted ubunto from a drive, and trying it on my laptop, haven't installed it or anything.....

i connect my HDD, then where should i look ?

**Comment by themaddoctor** [comment]
Please read the PDF. You have to open a terminal app.


On 3/9/18, Ahmed Kamal Pasha <notifications@github.com> wrote:
> where exactly do I run this code, I just booted ubunto from a drive, and
> trying it on my laptop, haven't installed it or anything.....
>
> i connect my HDD, then where should i look ?
>
> --
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/themaddoctor/linux-mybook-tools/issues/15#issuecomment-371777476


-- 
Thomas Kaeding

my public key: http://pgp.mit.edu/pks/lookup?op=get&search=0x81B0FCA32599BE6F

**Comment by ahmedkpasha** [comment]
Yeah, about that, i cant find that PDF file, I'm terribly new at this Thomas, and appreciate you talking to me at all about this issue. kindly guide me a bit.

**Comment by mrthielo** [comment]
He means[ this PDF file](https://github.com/themaddoctor/linux-mybook-tools/blob/master/Mounting%20encrypted%20WD%20disks%20in%20linux%200.5.1.pdf).

Wish you luck.

Marcelo

**Comment by inzainia** [comment]
Is this recoverable? or is there at least a good chance some data will be recoverable? I had the exact issue, except i originally had 2 partitions on the drive, removed it from the wd case, stuck it in pc, and re partitioned into 1 when i got the "this drive needs to be initialized to work in windows" crap.

I'm not asking for a guarantee, BUT if i spend the time and learn the linux stuff (i know almost nothing) is there a decent chance i can decrypt the drive, into an image, or whatever, and then deep scan it or whatever and recover the stuff i lost? any info would be appreciated before i delve into this. 

if it helps, i have vmware pro with linux mint, it sees the drive(bottom right), but as ive learned from reading here, it will never successfully mount the drive, until i can get a decrypted image off it,

**Comment by inzainia** [comment]
yes sir, unfortunately, my issue happened 2 years or so ago. the drive case fried, i tore it apart, and figured id try the bare drive in the windows pc, at that time, i had no idea WD added in hardware encryptors, as i never installed the software to password protect it. i just bought it and used it as a normal drive. 3 weeks ago i stumbled upon it while cleaning some old boxes out, and decided to see if i could recover anything.  so i am going to spend some of the next couple days trying to get the info you mentioned above. i will send it to ya as soon as i can get it. thanks so much for providing the syntax for the commands there, so that i didnt have to figure that out lol. every bit helps. and it is appreciated.
